source/release_notes/4.0.28-13.rst

```diff
@@ -0,0 +1,20 @@
+.. _PSMDB-4.0.28-23:
+
+================================================================================
+*Percona Server for MongoDB* 4.0.28-23
+================================================================================
+
+:Date: February 15, 2022
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.0/install/index.html>`_
+
+Percona Server for MongoDB 4.0.28 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.0.28 Community Edition.
+
+It is rebased on MongoDB 4.0.28 Community Edition and supports MongoDB 4.0.28 protocols and drivers.
```

Comment:
We have always used "It is based ..." wording. Now it is "rebased". Is this an intended change?

Reply:
Yep. Peer suggested this wording for cases when we introduce no changes/bug fixes of our own. Yet since we do, I removed it

Reply:
Ok